### ISSUE TITLE
Process keeps dying when trying to train model

### ISSUE THREAD

**Issue by cmeister747** [shlurbee/dmrs-text-generation-naacl2019#2]
I successfully preprocessed the data and am now trying to train the text generator model. I'm running 
`python OpenNMT-py/train.py -data data/opennmt         -layers 2 -dropout 0.5         -word_vec_size 250 -batch_type sents -max_grad_norm 5 -param_init_glorot         -encoder_type brnn -decoder_type rnn -rnn_type LSTM -rnn_size 400         -save_model models/v12         -learning_rate 0.001 -start_decay_at 25 -opt adam -epochs 40 -gpuid 0 -batch_size 12 > logs/train_v12.log` which is essentially just #12 in run_experiments.sh with the batch size parameter added. After about 10 minutes though, training stops and just prints "Killed" (no signal reported or anything). The problem continues even though I've shrunk the batch size to 12. I'm running this on a pretty large server so I feel like it shouldn't be a memory issue. Do you have any idea what might be going on?

**Comment by cmeister747** [comment]
Alternatively, would you guys be willing to share some of the pretrained model files? :)
